script.js

```diff
@@ -0,0 +1,315 @@
+    'Ctrl', 'Win', 'Alt', ' ', 'Alt', '←', '↓', '→', 'Ctrl'
+  ];
+
+  const keysEngShift = [
+    '~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+', 'Backspace',
+    'Tab', 'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', '{', '}', '|', 'Del',
+    'CapsLock', 'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ':', '"', 'Enter',
+    'Shift', 'Z', 'X', 'C', 'V', 'B', 'N', 'M', '<', '>', '?', '↑', 'Shift',
+    'Ctrl', 'Win', 'Alt', ' ', 'Alt', '←', '↓', '→', 'Ctrl'
+  ];
+
+  const keysRus = [
+    'ё', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+    'Tab', 'й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ', '\\', 'Del',
+    'CapsLock', 'ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э', 'Enter',
+    'Shift', 'я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю', '.', '↑', 'Shift',
+    'Ctrl', 'Win', 'Alt', ' ', 'Alt', '←', '↓', '→', 'Ctrl'
+  ];
+
+  const keysRusShift = [
+    'Ё', '!', '"', '№', ';', '%', ':', '?', '*', '(', ')', '_', '+', 'Backspace',
+    'Tab', 'Й', 'Ц', 'У', 'К', 'Е', 'Н', 'Г', 'Ш', 'Щ', 'З', 'Х', 'Ъ', '/', 'Del',
+    'CapsLock', 'Ф', 'Ы', 'В', 'А', 'П', 'Р', 'О', 'Л', 'Д', 'Ж', 'Э', 'Enter',
+    'Shift', 'Я', 'Ч', 'С', 'М', 'И', 'Т', 'Ь', 'Б', 'Ю', ',', '↑', 'Shift',
+    'Ctrl', 'Win', 'Alt', ' ', 'Alt', '←', '↓', '→', 'Ctrl'
+  ];
+
+  const body = document.querySelector('body');
+  const keyboardContainer = document.createElement('div');
+  const entryField = document.createElement('textarea');
+  const keyboard = document.createElement('div');
+  const keyboardTitle = document.createElement('h1');
+  const description = document.createElement('p');
+  const language = document.createElement('p');
+
+  keyboardTitle.innerText = 'Виртуальная клавиатура';
+  description.innerText = 'Клавиатура создана в операционной системе Windows';
+  language.innerText = 'Для переключения языка комбинация: левыe ctrl + alt';
+  entryField.setAttribute('readonly', 'readonly');
+
+  //Add documents to the DOM
+
+  function appendElement(elementName, className, elementParent) {
+    elementName.classList.add(className);
+    elementParent.appendChild(elementName);
+  }
+  appendElement(keyboardContainer, 'keyboard-container', body);
+  appendElement(keyboardTitle, 'keyboard-title', keyboardContainer);
+  appendElement(entryField, 'entry-field', keyboardContainer);
+  appendElement(keyboard, 'keyboard', keyboardContainer);
+  appendElement(description, 'description', keyboardContainer);
+  appendElement(language, 'language', keyboardContainer);
+
+  // Save last used lang in the storage
+
+  function setLastLang(){
+    let lang = 'eng';
+    if(keyboard.classList.contains('eng')){
+      lang = 'eng';
+      localStorage.setItem('lang', 'eng');
+    }else if(keyboard.classList.contains('rus')){
+      lang = 'rus';
+      localStorage.setItem('lang', 'rus');
+    }
+  }
+
+  // Get last used lang from the storage
+
+  function getLastLang(){
```

Comment:
Confusing variable naming. It doesn't behave as expected by it's name.

Reply:
`getLastLang()` renamed to `getLangFromStorage()`

Reply:
It's still not good name for this function.
Look, this name starts with **_get_**, so I'm expecting that this function will return me lang, but it returns `undefined`. And I should to go to this function and figure out what code inside does.
The best name of function should describe what function does. Moreover, when you trying to find convenient name, you can find how to refactor your function. In best case your function should do actualy what it's name describing and do only one thing.
I'm suggesting to name it like that: `updateKeyboardLayoutAccordingSelectedLang`